llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -8932,6 +8936,20 @@ VPRecipeBuilder::tryToCreatePartialReduction(Instruction *Reduction,
     std::swap(BinOp, Accumulator);
 
   unsigned ReductionOpcode = Reduction->getOpcode();
+  if (ReductionOpcode == Instruction::Sub) {
+    VPBasicBlock *ParentBlock = Builder.getInsertBlock();
+    if (!ParentBlock)
+      return nullptr;
```

Comment:
Can the insert block ever be null?  I'd expect it to always be set at this point?